For some reason, this line makes doxygen-1.8.6 eat all memory and segfault

--- doc/Doxyfile.extra.in~	2015-02-25 22:37:08.000000000 +0100
+++ doc/Doxyfile.extra.in	2015-04-29 10:40:56.138142749 +0200
@@ -1,4 +1,4 @@
 INPUT = @CMAKE_SOURCE_DIR@/include \
       @CMAKE_SOURCE_DIR@/doc
 
-TAGFILES = @HPP_MODEL_DOXYGENDOCDIR@/hpp-model.doxytag=@HPP_MODEL_DOXYGENDOCDIR@
+# TAGFILES = @HPP_MODEL_DOXYGENDOCDIR@/hpp-model.doxytag=@HPP_MODEL_DOXYGENDOCDIR@
